### ISSUE TITLE
Unexpected <

### ISSUE THREAD

**Issue by HDVinnie** [shufo/blade-formatter#8]
Im getting some odd errors in some views

```
{ path: 'resources/views/bonus/store.blade.php',
  message: 'Parse Error : syntax error, unexpected \'<\' on line 55\n\u001b[0m \u001b[90m 53 | \u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 54 | \u001b[39m                    \u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp \u001b[90m//@foreach ($downloadOptions as $d => $dO)\u001b[39m\u001b[0m\n\u001b[0m\u001b[31m\u001b[1m>\u001b[22m\u001b[39m\u001b[90m 55 | \u001b[39m                        \u001b[33m<\u001b[39m\u001b[33mtr\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m    | \u001b[39m                       \u001b[31m\u001b[1m^\u001b[22m\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 56 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp  $dO[\u001b[32m\'description\'\u001b[39m]  \u001b[33m?\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m/\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 57 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp  $dO[\u001b[32m\'cost\'\u001b[39m]  \u001b[33m?\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m/\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 58 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m',
  error: 
   { SyntaxError: Parse Error : syntax error, unexpected '<' on line 55
  53 | 
  54 |                     <?php //@foreach ($downloadOptions as $d => $dO)
> 55 |                         <tr>
     |                        ^
  56 |                             <td><?php  $dO['description']  ?></td>
  57 |                             <td><?php  $dO['cost']  ?></td>
  58 |                             <td>
    at parser.raiseError (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser.js:319:17)
    at parser.error (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser.js:364:15)
    at parser.read_expr_item (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser/expr.js:446:12)
    at parser.read_expr (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser/expr.js:12:19)
    at parser.read_statement (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser/statement.js:383:27)
    at parser.read_top_statement (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser/statement.js:71:21)
    at parser.read_start (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser/main.js:18:19)
    at parser.parse (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/parser.js:277:23)
    at engine.parseCode (/usr/lib/node_modules/blade-formatter/node_modules/php-parser/src/index.js:167:22)
    at Object.parse (/usr/lib/node_modules/blade-formatter/node_modules/@prettier/plugin-php/src/parser.js:30:57)
     fileName: 'eval',
     loc: { start: [Object] },
     codeFrame: '\u001b[0m \u001b[90m 53 | \u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 54 | \u001b[39m                    \u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp \u001b[90m//@foreach ($downloadOptions as $d => $dO)\u001b[39m\u001b[0m\n\u001b[0m\u001b[31m\u001b[1m>\u001b[22m\u001b[39m\u001b[90m 55 | \u001b[39m                        \u001b[33m<\u001b[39m\u001b[33mtr\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m    | \u001b[39m                       \u001b[31m\u001b[1m^\u001b[22m\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 56 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp  $dO[\u001b[32m\'description\'\u001b[39m]  \u001b[33m?\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m/\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 57 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp  $dO[\u001b[32m\'cost\'\u001b[39m]  \u001b[33m?\u001b[39m\u001b[33m>\u001b[39m\u001b[33m<\u001b[39m\u001b[33m/\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 58 | \u001b[39m                            \u001b[33m<\u001b[39m\u001b[33mtd\u001b[39m\u001b[33m>\u001b[39m\u001b[0m' } }
{ path: 'resources/views/playlist/show.blade.php',
  message: 'Parse Error : syntax error, unexpected \'<\' on line 213\n\u001b[0m \u001b[90m 211 | \u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 212 | \u001b[39m\t\t\u001b[33m<\u001b[39m\u001b[33m?\u001b[39mphp \u001b[90m//<div class="block">\u001b[39m\u001b[0m\n\u001b[0m\u001b[31m\u001b[1m>\u001b[22m\u001b[39m\u001b[90m 213 | \u001b[39m\t\t\t\t\u001b[33m<\u001b[39m\u001b[33mdiv\u001b[39m \u001b[36mclass\u001b[39m\u001b[33m=\u001b[39m\u001b[32m"row"\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m     | \u001b[39m\t\t\t\u001b[31m\u001b[1m^\u001b[22m\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 214 | \u001b[39m\t\t\t\t\t\u001b[33m<\u001b[39m\u001b[33mdiv\u001b[39m \u001b[36mclass\u001b[39m\u001b[33m=\u001b[39m\u001b[32m"col-xs-6 col-sm-4 col-md-3"\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 215 | \u001b[39m\t\t\t\t\t\t\u001b[33m<\u001b[39m\u001b[33mdiv\u001b[39m \u001b[36mclass\u001b[39m\u001b[33m=\u001b[39m\u001b[32m"image-box text-center mb-20"\u001b[39m\u001b[33m>\u001b[39m\u001b[0m\n\u001b[0m \u001b[90m 216 | \u001b[39m\t\t\t\t\t\t\t\u001b[33m<\u001b[39m\u001b[33mdiv\u001b[39m \u001b[36mclass\u001b[39m\u001b[33m=\u001b[39m\u001b[32m"overlay-container"\u001b[39m\u001b[33m>\u001b[39m\u001b[0m',
```

The `<` is not unexpected though.

**Comment by palpalani** [comment]
Yes, I am also getting the same error when there is a multiline comment block like `{{-- --}}`.

**Comment by shufo** [comment]
@HDVinnie 
Thanks for feedback!
Could you provide input and expected output?

**Comment by HDVinnie** [comment]
Its due to blade commenting.

```
                    {{--@foreach ($downloadOptions as $d => $dO)
                        <tr>
                            <td>{{ $dO['description'] }}</td>
                            <td>{{ $dO['cost'] }}</td>
                            <td>
                                <form method="POST" action="{{ route('bonusexchange', ['id' => $dO['id']]) }}">
                                    @csrf
                                    <button type="submit" class="btn btn-sm btn-info btn-exchange">@lang('bon.exchange')</button>
                                </form>
                            </td>
                        </tr>
                    @endforeach--}}
```


Here is input:
https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/resources/views/bonus/store.blade.php

**Comment by lanphan** [comment]
Hi all,
Is there any progress on this problem? I have same problem.

My input:
```
{{-- <li class="nav-item">
                            <a href="{{ route("configuration.vendor-complaint-entry.index") }}" class="nav-link
        {{ request()->is('configuration/vendor-complaint-entry*') ? 'active' : '' }}">
        <p>
            <span>{{ trans('cruds.vendorComplaintEntry.title') }}</span>
        </p>
        </a>
</li> --}}
```

**Comment by shufo** [comment]
I just published [1.2.2](https://www.npmjs.com/package/blade-formatter/v/1.2.2) and fixes this behaviour.
